src/color.rs

```diff
@@ -223,6 +223,20 @@ impl Color {
         a: NV_ONE,
     };
 
+    /// Creates a new color from 4 components.
+    ///
+    /// # Safety
+    ///
+    /// All values must be in 0..=1 range.
+    pub const unsafe fn from_unchecked_rgba(r: f32, g: f32, b: f32, a: f32) -> Self {
+        Color {
+            r: NormalizedF32::new_unchecked(r),
```

Comment:
In theory, this could use a safety comment. In practise, the safety comment is the "unit" safety comment that preconditions are guaranteed by caller.